src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/DecideFtpaApplicationOutcomeType.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.entities;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum DecideFtpaApplicationOutcomeType {
```

Comment:
Here we are reusing the ResidentJudgeOutcomeType class I think that you can delete this one